apis/vshn/v1/dbaas_vshn_keycloak.go

```diff
@@ -278,7 +278,7 @@ func (v *VSHNKeycloak) GetFullMaintenanceSchedule() VSHNDBaaSMaintenanceSchedule
 	return schedule
 }
 
-// GetAllowedNamespaces returns allowed namespaces to access this service
-func (v *VSHNKeycloak) GetAllowedNamespaces() []string {
-	return v.Spec.Parameters.Security.AllowedNamespaces
+// GetSecurity returns the security object this service
```

Comment:
Please add this to the API template as well. The next guy who creates a new service will thank you. :D

It's under `hack/bootstrap/template`

Reply:
I added it, now sure if it's right, please check